app/app/Http/Controllers/LocationRatingController.php

```diff
@@ -115,7 +115,7 @@ public function show(int $location_id, int $question_category_id = null)
 			return redirect()->intended('/signin');
 		}
 	   $location = Location::find($location_id);
-	   $question_categories = QuestionCategory::with('questions')->get();
+	   $question_categories = QuestionCategory::with('questions')->orderBy('name','ASC')->get();
```

Comment:
There's a couple extra changes needed with the navigation.

When the rating feature was added, the "Next" button would navigate to the next question category.  "Submit" should be at the last category.  The old way of calculating the next category needs to be updated to match the new order.

Here is a screenshot showing "Submit" for the "Amenities" category when that button should now be in "Public Areas" since that's the last category:
![image](https://user-images.githubusercontent.com/11838191/33108432-1604904c-cf0a-11e7-8efc-aab5f7f1e05e.png)

You might want to consult http://app.accesslocator.com to see how the "Next" navigation matches with the category list order before these changes.

I think this line is the thing to change in LocationRatingController.php:
```
$next_question_category = QuestionCategory::where('id', '>', $question_category_id)
```
You probably want to compare 'name' instead of 'id' with the current category's name.  You may also want to orderBy or filter to get the minimum-named question category that is still larger than the current question category's name.